pyqtgraph/functions.py

```diff
@@ -1765,7 +1765,7 @@ def gaussianFilter(data, sigma):
     return filtered + baseline
     
     
-def downsample(data, n, axis=0, xvals='subsample'):
+def downsample(data, n, axis=0, xvals='subsample', has_nans=False):
```

Comment:
Can you make `has_nans` be keyword only?    This can be done via `..., xvals='subsamble', *, has_nans=False):`

Reply:
This has been resolved.